tests/src/test/scala/whisk/core/controller/test/MetaApiTests.scala

```diff
@@ -64,16 +65,13 @@ import whisk.http.Messages
  * "using Specification DSL to write unit tests, as in should, must, not, be"
  * "using Specs2RouteTest DSL to chain HTTP requests for unit testing, as in ~>"
  */
-@RunWith(classOf[JUnitRunner])
-class MetaApiTests extends ControllerTestCommon with WhiskMetaApi with BeforeAndAfterEach {
-
-    override val apipath = "api"
-    override val apiversion = "v1"
```

Comment:
Having trouble overriding these variables at the proper time during instantiation, specifically for webInvokePath. Hence the lazy val technique.